### DISCUSSION TITLE
How to sign in Web UI?

### DISCUSSION THREAD

**Discussion by BlackBirderMan** [wg-easy/wg-easy#234] [Q&A]
I done all actions from instruction, but http://0.0.0.0:51821 is not opening.
Also i changed 0.0.0.0 to my server's ip, but nothing changed.
What am i doing wrong?

**Comment by eandersons** [comment]
Did you run WireGuard Easy using `docker run` or Docker Compose?
In either case the full `docker run` command or content of the used `docker-compose.yml` would help to spot issues or at least rule out some possibilities.

Also, are you trying to access <http://0.0.0.0:51821> from the same machine where WireGuard Easy is running or from other computer in the same LAN?
If the latter, then most probably server's on which WireGuard Easy is running firewall is blocking incoming requests for the port 51821.

**Reply by BlackBirderMan** [reply]
i do all from this
[](https://github.com/WeeJeWel/wg-easy/discussions/234#3-sponsor)2. Run WireGuard Easy

To automatically install & run wg-easy, simply run:

$ docker run -d \
  --name=wg-easy \
  -e WG_HOST=🚨YOUR_SERVER_IP \
  -e PASSWORD=🚨YOUR_ADMIN_PASSWORD \
  -v ~/.wg-easy:/etc/wireguard \
  -p 51820:51820/udp \
  -p 51821:51821/tcp \
  --cap-add=NET_ADMIN \
  --cap-add=SYS_MODULE \
  --sysctl="net.ipv4.conf.all.src_valid_mark=1" \
  --sysctl="net.ipv4.ip_forward=1" \
  --restart unless-stopped \
  weejewel/wg-easy
💡 Replace YOUR_SERVER_IP with your WAN IP, or a Dynamic DNS hostname.

💡 Replace YOUR_ADMIN_PASSWORD with a password to log in on the Web UI.
The Web UI will now be available on http://0.0.0.0:51821.

💡 Your configuration files will be saved in ~/.wg-easy

**Reply by eandersons** [reply]
> Also, are you trying to access <http://0.0.0.0:51821> from the same machine where WireGuard Easy is running or from other computer in the same LAN?

What about this?

Also, what value did you use for the environment variable `WG_HOST`: `0.0.0.0`, IP address of the machine in the LAN or the public IP address?

**Reply by BlackBirderMan** [reply]
> > Also, are you trying to access <http://0.0.0.0:51821> from the same machine where WireGuard Easy is running or from other computer in the same LAN?
> 
> 
> 
> What about this?
> 
> 
> 
> Also, what value did you use for the environment variable `WG_HOST`: `0.0.0.0`, IP address of the machine in the LAN or the public IP address?

I bougth vps and do 1 an 2 steps in the instruction. That's all. I'm trying to connect on my computer.

**Reply by eandersons** [reply]
I am trying to help you, but if you are not providing information, I cannot do it.
The devil is in the details.

If you are using VPS and you are trying to access WireGuard Easy UI from outside, then I am betting that firewall is blocking incoming requests for the port 51821 (something like that I already mentioned in my [first post](#discussioncomment-2435752)).

An alternative solution to allowing access to the port 51821 in the firewall would be to use a proxy server as most probably ports 80 and 443 for your VPS instance are already open to the outside world.

**Reply by BlackBirderMan** [reply]
> I am trying to help you, but if you are not providing information, I cannot do it. The devil is in the details.
> 
> If you are using VPS and you are trying to access WireGuard Easy UI from outside, then I am betting that firewall is blocking incoming requests for the port 51821 (something like that I already mentioned in my [first post](#discussioncomment-2435752)).
> 
> An alternative solution to allowing access to the port 51821 in the firewall would be to use a proxy server as most probably ports 80 and 443 for your VPS instance are already open to the outside world.

what if i change this commands
-p 51820:51820/udp \
  -p 51821:51821/tcp \

to this
-p 80:80/udp \
  -p 443:443/tcp \
and will try to connect to 0.0.0.0:80 or 443?
and maybe i need to install original wireguard on vps before installing wg easy?

**Reply by eandersons** [reply]
First of all, there is no need to install WireGuard on host.

You could try to use ports 80 and 443, but I doubt that it will work, because:

- I doubt that ports 80 and 443 on VPS usually are opened for UDP traffic;
- most probably a SSL certificate would be needed to connect to the TCP 443 port,

but I am not completely sure about either.
You will have to find it out yourself by trying.
Or you could just ask your VPS provider to open WireGuard Easy specific ports.

A side note: you could use some online port checker tool to test if specific TCP ports are open on your VPS (for example: <https://dnschecker.org/port-scanner.php>).
UDP port checking is more complicated.

**Comment by a-gt** [comment]
Because you are hosting wg-easy on a VPS, you won't be able to connect to it via 0.0.0.0. You need to connect to it using the VPS ip.

**Comment by BlackBirderMan** [comment]
> Because you are hosting wg-easy on a VPS, you won't be able to connect to it via 0.0.0.0. You need to connect to it using the VPS ip.

Of course i changed zero's ip to my vps ip, but nothing changed.
